src/CC-BY-ND-2.5.xml

```diff
@@ -7,7 +7,7 @@
       </crossRefs>
     <text>
       <titleText>
-         <p>Creative Commons Attribution-NoDerivs 2.5</p>
+         <p>Creative Commons Attribution-NoDerivs 2.5 Generic</p>
```

Comment:
One more thing also confuse me, the link https://creativecommons.org/licenses/by-nd/2.5/legalcode license name is "Attribution-NoDerivs 2.5 Generic" but in spdx named "Creative Commons Attribution-NoDerivs 2.5 Generic", should we also remove the leading word "Creative Commons"?

```suggestion
         <p>Attribution-NoDerivs 2.5 Generic</p>
```

Reply:
And in https://spdx.org/licenses/CC-BY-ND-2.5.html I find out the "full name" have "Creative Commons" keyword but https://creativecommons.org/licenses/by-nd/2.5/legalcode license name is "Attribution-NoDerivs 2.5 Generic", should we also remove "Creative Commons" for the name
Or I want to know in which case we will add a prefix like "Creative Commons" in the title?